### ISSUE TITLE
Bundler/OrderedGems fails on empty conditionals

### ISSUE THREAD

**Issue by dblock** [bbatsov/rubocop#5128]
OrderedGems fails on conditionals that evaluate to empty in Gemfile, for [example](https://github.com/slack-ruby/slack-ruby-client/blob/master/Gemfile#L5):

```ruby
gem ENV['CONCURRENCY'], require: false if ENV.key?('CONCURRENCY')
```
--------

## Expected behavior

It should probably ignore this line or work on an evaluated result.

## Actual behavior

```
An error occurred while Bundler/OrderedGems cop was inspecting /Users/dblock/source/slack/slack-ruby-client/dblock/Gemfile.
```

## Steps to reproduce the problem

Add the line above into a Gemfile, run `rubocop`.

## RuboCop version

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.2, running on ruby 2.4.1 x86_64-darwin16)
```

## Workaround

Oddly no `enable` required.

```
gem ENV['CONCURRENCY'], require: false if ENV.key?('CONCURRENCY')
# rubocop:enable Bundler/OrderedGems
```
